src/common/post.ts

```diff
@@ -329,3 +344,125 @@ export const insertCodeSnippetsFromUrl = async ({
     throw err;
   }
 };
+
+type ModeratedPostCdc = ChangeObject<SourcePostModeration>;
+
+export const updateModeratedPost = async (
+  con: ConnectionManager,
+  moderated: ModeratedPostCdc,
+): Promise<ModeratedPostCdc | null> => {
+  if (!moderated?.postId) {
+    logger.error('unable to update moderated post', { moderated });
+    return null;
+  }
+
+  const postParam = { id: moderated.postId };
+  const repo = con.getRepository(Post);
+
+  await repo.findOneOrFail({ where: { id: moderated.postId } });
```

Comment:
This is just for failing if doesn't exist? (maybe add explicit id select to narrow scope)

Reply:
I'm not sure I understand. We have `id: moderated.postId` already right?

Reply:
Oh, didn't know you actually pull everything if you don't specify. Yeah, sure let's do that.